### ISSUE TITLE
Fails on RHEL7 AWS, packer install

### ISSUE THREAD

**Issue by gloriajw-devmynd** [geerlingguy/ansible-role-clamav#7]
amazon-ebs: failed: [default] (item=[u'clamav', u'clamav-update', u'clamav-scanner-systemd']) => {
    amazon-ebs:     "changed": false,
    amazon-ebs:     "invocation": {
    amazon-ebs:         "module_args": {
    amazon-ebs:             "allow_downgrade": false,
    amazon-ebs:             "bugfix": false,
    amazon-ebs:             "conf_file": null,
    amazon-ebs:             "disable_gpg_check": false,
    amazon-ebs:             "disable_plugin": [],
    amazon-ebs:             "disablerepo": null,
    amazon-ebs:             "enable_plugin": [],
    amazon-ebs:             "enablerepo": null,
    amazon-ebs:             "exclude": null,
    amazon-ebs:             "install_repoquery": true,
    amazon-ebs:             "installroot": "/",
    amazon-ebs:             "list": null,
    amazon-ebs:             "name": [
    amazon-ebs:                 "clamav",
    amazon-ebs:                 "clamav-update",
    amazon-ebs:                 "clamav-scanner-systemd"
    amazon-ebs:             ],
    amazon-ebs:             "security": false,
    amazon-ebs:             "skip_broken": false,
    amazon-ebs:             "state": "latest",
    amazon-ebs:             "update_cache": false,
    amazon-ebs:             "update_only": false,
    amazon-ebs:             "validate_certs": true
    amazon-ebs:         }
    amazon-ebs:     },
    amazon-ebs:     "item": [
    amazon-ebs:         "clamav",
    amazon-ebs:         "clamav-update",
    amazon-ebs:         "clamav-scanner-systemd"
    amazon-ebs:     ],
    amazon-ebs:     "msg": "No package matching 'clamav' found available, installed or updated",
    amazon-ebs:     "rc": 126,
    amazon-ebs:     "results": [
    amazon-ebs:         "No package matching 'clamav' found available, installed or updated"
    amazon-ebs:     ]
    amazon-ebs: }
    amazon-ebs: 	to retry, use: --limit @/opt/mirs/mirs-ansible/base.retry
    amazon-ebs:
    amazon-ebs: PLAY RECAP *********************************************************************
    amazon-ebs: default                    : ok=21   changed=12   unreachable=0    failed=1

**Comment by gloriajw-devmynd** [comment]
Solved it after much trial and error:
yum: name={{ item }} state=latest disable_gpg_check=yes
https://github.com/ComplianceAsCode/content/issues/1596

**Comment by gloriajw-devmynd** [comment]
Turning off the GPG key validation helps it along a bit further, but installation still fails with the same error.

**Comment by gloriajw-devmynd** [comment]
Thanks for the info. I am using AWS RHEL7. I finally made good progress by doing this in the EPEL ansible config:

`  lineinfile:
     path: /etc/yum.repos.d/epel.repo
     line: 'includepkgs=*'
     state: present
     insertafter: 'repo=epel-7'
`
It was not easy to find out which packages were essential for clamav, so I installed them all. Now clamav is present. The first change I applied to get past the initial gpg check turns out to be unnecessary. Installing all EPEL packages solved the problem.

Thanks for your guidance.
